README.md

```diff
@@ -186,7 +186,8 @@ python tools/eval.py -n  yolox-s -c yolox_s.pth -b 1 -d 1 --conf 0.001 --fp16 --
 <details>
 <summary>Tutorials</summary>
 
-*  [Training on custom data](docs/train_custom_data.md).
+*  [Training on custom data](docs/train_custom_data.md)
+*  [Controlling training image size](docs/control_training_image_size.md)
```

Comment:
Using "Modify" might be better that"Control" ?

Reply:
This tutorial covers two topics:

1. using the default size
2. using modified sizes

''Modify'' only stands for case 2. So I think ''control'' is a better description. What do you think?